lib/Model/Listing/AbstractListing.php

```diff
@@ -310,8 +310,8 @@ public function getCondition()
                 // If there is not a placeholder, ignore value!
                 if (!$value['ignore-value']) {
                     if (is_array($value['value'])) {
-                        foreach ($value['value'] as $k => $v) {
-                            $params[$k] = $v;
+                        foreach ($value['value'] as $v) {
+                            $params[] = $v;
```

Comment:
@t-zilla I'm afraid, this change will break named params.

Reply:
@dvesh3 Good point, I accounted for it in the next commit